crawlers/collectionsScanner.ts

```diff
@@ -1,10 +1,11 @@
+import pino, { Logger } from 'pino';
 import { OpalAPI } from 'lib/providerAPI/bridgeProviderAPI/concreate/opalAPI';
 import { TestnetAPI } from 'lib/providerAPI/bridgeProviderAPI/concreate/testnetAPI';
 import { Sequelize } from 'sequelize/types';
-import pino, { Logger } from 'pino';
+import { ICollectionDB } from '../lib/collection/collectionDB.interface';
```

Comment:
Maybe you mean database entity?
ICollectionEntity?

Reply:
Ok, let it be `ICollectionDbEntity`